sample-config-files/defender-config.yaml

```
@@ -42,4 +42,8 @@ Defender:
       ExcludedGroups:
         -
       ExcludedDomains:
-        -
\ No newline at end of file
+        -
+  MS.DEFENDER.2.1v1 : &UserImpersonationProtection
+    SensitiveAccounts:
+      IncludedUsers:
+        - johndoe@random.example.com
```

Recommend changing to match the actual output format Microsoft uses. When you add a user to targeted protection, you are required to indicate both their display name as well as their email, and the output is formatted like this: `Display name;email`.

```suggestion
        - John Doe;johndoe@random.example.com
```

---

Fix pushed, please confirm

---

I see that you updated the unit tests. Could you also update the example config.yaml file?

---

Whoops, completely missed that. Fixed